governance/votes/Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,202 @@
+date: 2023-07-12T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] BlockTower Andromeda Upgrade, Smart Burn Engine Deployment, Keeper Job Updates, Scope Defined Parameter Changes, Delegate Compensation, Ecosystem Actor Funding, Spark Protocol Proxy Spell Execution - July 12, 2023
+
+The Governance Facilitators and Sidestream, PullUp, and dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes and additions** will occur within the Maker Protocol:
+- A new multiswap output conduit for BlockTower Andromeda (RWA015-A) will be deloyed, as detailed below.
+- The new Smart Burn Engine will be deployed, as detailed below.
+- Keeper jobs will be updated, as detailed below.
+- Multiple parameter changes per the Stability Scope will be made, as detailed below.
+- Delegate compensation for June 2023 will be distributed, as detailed below.
+- Funding stream setup and housekeeping for two Ecosystem Actors will be carried out, as detailed below.
+- A proxy spell for Spark Protocol will be executed, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### BlockTower Andromeda (RWA015-A) Update
+
+As per the original [technical assessment](https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974) a multiswap Output Conduit was initially proposed for this vault type. This will allow the `operator` role to choose which PSM is used to obtain redeemable stablecoins in return for the DAI drawn from the vault. At present the vault is set up to only use the USDP PSM as a compromise while the multiswap tool was developed. The multiswap Output Conduit is now audited and deployed, and the following actions will be carried out if this executive proposal passes. 
```

Comment:
DAI vs Dai throughout exec

Reply:
most of these are intentional - `Dai` is the name of a currency, `DAI` is a unit of measurement. This one is an error though